src/document.h

```diff
@@ -570,7 +568,7 @@ struct Document {
         ResetFont();
         dc.SetUserScale(1, 1);
         curdrawroot = WalkPath(drawpath);
-        int psb = curdrawroot == rootgrid ? 0 : curdrawroot->MinRelsize();
+        int psb = curdrawroot == rootgrid.get() ? 0 : curdrawroot->MinRelsize();

```

Comment:
We need to call `.get()` because curdrawroot is `Cell *` and rootgrid is `unique_ptr<Cell>`.

Reply:
https://en.cppreference.com/w/cpp/memory/unique_ptr/operator_bool

Reply:
I think the precedence order here is:
`int psb = ((curdrawroot == rootgrid.get()) ? 0 : curdrawroot->MinRelsize());`
Is this correct? If yes, I cannot compare `curdrawroot` and `rootgrid` directly?

Reply:
Ah, did not see. You're right!